app/controllers/admin_comment_controller.rb

```diff
@@ -38,24 +38,39 @@ def update
     end
 
     if @comment.update(comment_params)
-      update_type = if comment_hidden?
-        'hide_comment'
-      else
-        'edit_comment'
+      if @comment.previous_changes[:body]
```

Comment:
One really minor improvement we could make here is extracting the details to private methods:

```ruby
log_edit_comment(@comment) if @comment.previous_changes[:body]
log_hide_comment(@comment) if @comment.previous_changes[:visible]
log_report_comment(@comment) if @comment.previous_changes[:attention_requested]
```